### PR TITLE
Minor Finetuning Scheduler Tutorial Update

### PR DESCRIPTION
**Pull request by speediedan**
## Before submitting

- [X] Was this **discussed/approved** via a Github issue? (no need for typos and docs improvements)
- [X] Did you make sure to **update the docs**?
- [X] Did you write any new **necessary tests**?

## What does this PR do?

Minor fix for stale finetuning scheduler link, fix for finetuning scheduler tutorial tag capitalization and minor tutorial update referencing additional extension features.

Note that this PR should be merged in addition to the [minor css enhancement PR](https://github.com/PyTorchLightning/lightning_sphinx_theme/pull/41) to allow the [submodule update PR](https://github.com/PyTorchLightning/pytorch-lightning/pull/13268) to succeed.

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: lightning_examples/finetuning-scheduler/finetuning-scheduler.py [hunk 1]
```diff
@@ -524,6 +529,8 @@ def configure_optimizers(self):
 # used in other pytorch-lightning tutorials) also work with FinetuningScheduler. Though the LR scheduler is theoretically
 # justified [(Loshchilov & Hutter, 2016)](#f4), the particular values provided here are primarily empircally driven.
 #
+# [FinetuningScheduler](https://finetuning-scheduler.readthedocs.io/en/stable/api/finetuning_scheduler.fts.html#finetuning_scheduler.fts.FinetuningScheduler) also supports LR scheduler
+# reinitialization in both explicit and implicit finetuning schedule modes. See the [advanced usage documentation](https://finetuning-scheduler.readthedocs.io/en/stable/advanced/lr_scheduler_reinitialization.html) for explanations and demonstration of the extensions' support for more complex requirements.
```
Comment (rohitgr7) [inline lightning_examples/finetuning-scheduler/finetuning-scheduler.py:533]:
```suggestion
# reinitialization in both explicit and implicit finetuning schedule modes. See the [advanced usage documentation](https://finetuning-scheduler.readthedocs.io/en/stable/advanced/lr_scheduler_reinitialization.html) for explanations and demonstration of the extension's support for more complex requirements.
```

**Review by rohitgr7 — APPROVED** [review]

File: lightning_examples/finetuning-scheduler/.meta.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 title: Finetuning Scheduler
 author: "[Dan Dale](https://github.com/speediedan)"
 created: 2021-11-29
-updated: 2022-05-10
+updated: 2022-06-10
 license: CC BY-SA
-build: 3
+build: 4
```
Comment (Borda) [inline lightning_examples/finetuning-scheduler/.meta.yml:6]:
```suggestion
build: 0
```
I know it is not described; this is just to trigger build is if there are no other changes in config nor script

**Review by Borda — APPROVED** [review]
